packages/teams-js/src/public/calendar.ts

```diff
@@ -106,4 +127,55 @@ export namespace calendar {
     /** The body content of the meeting. */
     content?: string;
   }
+
+  export interface JoinMeetingParams {
+    skypeTeamsMeetingUrl: string;
+    subject: string;
+    source: string;
+    conversationId: string;
+    messageId: string;
+    rootMessageId: string;
+    isBroadcastMeeting?: boolean | undefined;
+    t2BroadcastParticipantRoleDetails?: IT2BroadcastParticipantRoleDetails | undefined;
+  }
+
+  export interface IT2BroadcastParticipantRoleDetails {
```

Comment:
I don't think all this is needed.
Remove all the broadcast related items.

Reply:
Removed all the broadcast related details.